lib/questionnaires/view/item/answer/src/coding_answer_filler.dart

```diff
@@ -398,26 +391,41 @@ class _HorizontalCodingChoices extends AnswerInputControl<CodingAnswerModel> {
     super.focusNode,
   });
 
-  final List<Widget> choices;
+  final List<_CodingChoice> choices;
 
   @override
   Widget build(BuildContext context) {
     return Column(
       mainAxisSize: MainAxisSize.min,
-      crossAxisAlignment: CrossAxisAlignment.start,
+      crossAxisAlignment: CrossAxisAlignment.stretch,
       children: [
         _CodingChoiceDecorator(
           answerModel,
           focusNode: focusNode,
-          child: Row(
-            children: choices.map<Widget>(
-              (choice) {
-                return choice is _NullRadioChoice
-                    ? SizedBox(width: 96, child: choice)
-                    : Expanded(child: choice);
-              },
-            ).toList(growable: false),
-          ),
+          child:
+              QuestionnaireTheme.of(context).allowHorizontalCodingMultipleLines
+                  ? Wrap(
+                      spacing: 10.0,
+                      children: choices.map<Widget>((choice) {
+                        return IntrinsicWidth(
+                          child: Container(
+                            constraints: choice.answerOption == null
+                                ? const BoxConstraints(minWidth: 96)
+                                : null,
+                            child: choice,
+                          ),
+                        );
+                      }).toList(growable: false),
+                    )
+                  : Row(
+                      children: choices.map<Widget>(
+                        (choice) {
+                          return choice.answerOption == null
+                              ? SizedBox(width: 96, child: choice)
+                              : Expanded(child: choice);
+                        },
+                      ).toList(growable: false),
+                    ),
```

Comment:
@hanny-ph I think this change is fine, but I wonder if maybe there's some way to improve it, or if maybe we should always default to wrapping the choices maybe?

Reply:
> or if maybe we should always default to wrapping the choices maybe?

I was thinking of having the same pattern with `codingCheckboxChoiceBuilder` and `codingRadioChoiceBuilder`. Do you think that's better?

Reply:
@hanny-ph Yeah, I think that'd be better and simpler maybe.

Reply:
@fjwong I made some changes here: https://github.com/sujrd/faiadashu/pull/130/commits/ea93321c6c113b7ee7e49dcf4a1bf3e185b2ece4

Do you think it's also necessary to have vertical layout builder?

Reply:
@hanny-ph That's a good question. 

While we don't explicitly need it at the moment, it'd probably be better to add it now I think.

I'd say that, rather than having two layout builders (if it makes things easier that is), we could have only one that we can use in both places, and pass to it an additional argument indicating whether the orientation is vertical or horizontal. But if it's simpler to have the two builders, we could go that way as well.

Reply:
>  it's simpler to have the two builders, we could go that way as well.

Yeah, better to split this into two builders, because most likely they will have different implementations (Widget). Personally this looks cleaner.